app/components/file-details/summary/file-tags/index.scss

```diff
@@ -0,0 +1,7 @@
+.add-tag-btn {
+  height: 32px;
+}
+
+.add-tag-text-field-root {
```

Comment:
Unused style declaration.

Reply:
this is getting used here https://github.com/appknox/irene/pull/1228/files/1276ed7d4cd194ae28f2a05b6b4afc7b87a21a09#diff-56b49785a0a8d4d665ca4a314394e4d6d27c83d54188df105a3035bb0f60d8beR33